compiler/ballerina-backend-llvm/src/main/ballerina/src/compiler_backend_llvm/llvm_func_gen.bal

```diff
@@ -15,14 +15,14 @@
 // under the License.
 
 import ballerina/llvm;
-//import ballerina/io;
 import ballerina/bir;
 
 type FuncGenrator object {
     bir:Function func;
     llvm:LLVMValueRef? funcRef = ();
     llvm:LLVMModuleRef mod;
     map<llvm:LLVMValueRef>? localVarRefs = {};
```

Comment:
Not related to this PR, but why does this `localVarRefs` needs to be nil-able?

Reply:
I was under the assumption that there could be chances where local variables are non-existing in a function. Is that invalid?

Reply:
an empty map would represent that right?

Reply:
Yes! I will update.